client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -662,6 +663,10 @@
                     superdesk.intent('view', 'content');
                 };
 
+                $scope.isMediaType = function isMediaType(item) {
+                    return item && _.contains(['audio', 'video', 'picture'], item.type);
```

Comment:
this won't change during item editing, so it could be like ```$scope.isMediaType = _.contains(.., item.type);```
there is no need to do this iteration in every digest. or those could be just one time watches in template